frb_codegen/src/ir/ident.rs

```diff
@@ -1,9 +1,21 @@
 use convert_case::{Case, Casing};
 
-#[derive(Debug, Clone)]
+crate::ir! {
+#[no_serde]
 pub struct IrIdent {
     pub raw: String,
 }
+}
+
+#[cfg(feature = "serde")]
+impl serde::Serialize for IrIdent {
+    fn serialize<S>(&self, s: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        s.serialize_newtype_struct("IrIdent", &self.raw)
```

Comment:
uber-nit: why do we manually write down serialization? Is it possible we auto generate it (possibly with some annotations)